images/macos/software-report/SoftwareReport.Common.psm1

```diff
@@ -487,6 +487,11 @@ function Get-PowershellVersion {
     return $powershellVersion
 }
 
+function Get-SwigVersion {
+    $swigVersion = Run-Command "swig --version" -SuppressStderr
```

Comment:
What is the output of `swig --version`? Does it return only version or any other info?

Reply:
@kassannk have you tested it? There is no `--version` flag, there should be only one dash `-version`

Reply:
> @kassannk have you tested it? There is no `--version` flag, there should be only one dash `-version`

I have already fixed this in the case of windows and mac